src/SFA.DAS.EmployerCommitmentsV2.Web/Extensions/AgreementExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Linq;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Extensions
+{
+    public static class AgreementExtension
+    {   
+        public static bool HasSignedAgreement(this LegalEntity legalEntity, bool isTransfer)
```

Comment:
could this name be a bit more obvious as to what it is doing? `HasSignedMinimumRequiredAgreementVersion` or something like that

Reply:
done